puresasl/mechanisms.py

```diff
@@ -52,6 +53,8 @@ class Mechanism(object):
 
     def __init__(self, sasl, **props):
         self.sasl = sasl
+        self.qops = [QOP.AUTH]
+        self.qop = QOP.AUTH
```

Comment:
The intention, as you say below, is to add supported QOPs to mechanisms. I did fail to take into account client-selected QOPs when making this change - I'll get that fixed